### PR TITLE
Update delete and find commands, UserGuide and GUI

### PR DESCRIPTION
**Pull request by tobihy**
Find command: make partial word matches work

Find command only works for full word matches.

A partial-matching find is more user-friendly because users can find pets
by using words they think are in the pets' names.

Let's,
* update the search algorithm to use partial word matching
* edit tests to account for the new find command

### CODE REVIEW

File: copyright.txt [hunk 1]
```diff
@@ -1,9 +1,5 @@
 ﻿Some code adapted from http://code.makery.ch/library/javafx-8-tutorial/ by Marco Jakob
```
Comment (soloplxya) [inline copyright.txt:2]: Should we also consider including the source for `dog_icon.png` in this file?

**Review by soloplxya — COMMENTED** [review]
Everything looks good to me. Just one small thing to consider. LGTM!! 👍

**Comment by Dineshraj555** [comment]
LGTM as well! Nice work.

File: copyright.txt (hunk 1 above)
Comment (tobihy) [reply copyright.txt:2]: Good point, will include!

**Pull request merged**